src/components/List/RowLabel.js

```diff
@@ -103,9 +113,21 @@ const renderLabel = (label, variant) => {
         </LabelText>
       );
     case "caution":
-      return <StatusBadge label={label} color={themes.global.caution.dark} />;
+      return (
+        <StatusBadge
+          label={label}
+          color={themes.global.caution.dark}
+          style={styles.badge}
+        />
+      );
     case "neutral":
-      return <StatusBadge label={label} color={themes.global.onyx.light} />;
+      return (
+        <StatusBadge
+          label={label}
+          color={themes.global.onyx.light}
+          style={styles.badge}
```

Comment:
Why are we changing styles of a global component here? Perhaps it was missed in original implementation and the line-height should be changed in the badge component itself?

Reply:
I just wanted to reduce the height of the badges for list row component and nowhere else, but in order to maintain consistency within the entire library, I will fix this too.